README.md

```diff
@@ -77,7 +77,7 @@ As an example, this is the JSON bit which has to be added in the `target_overrid
         },
 ```
 
-<span> **Note:** You can get more informations about the configuration system in the [documentation](https://github.com/ARMmbed/mbed-os/blob/master/docs/config_system.md)</span>
+<span> **Note:** You can get more information about the configuration system in the [documentation](https://os.mbed.com/docs/latest/reference/configuration.html)</span>
```

Comment:
This link may be better in this context: https://os.mbed.com/docs/v5.6/tools/configuring-tools.html

Reply:
Yes, you are right, but how about we change your suggestion slightly to use "latest" : https://os.mbed.com/docs/latest/tools/configuring-tools.html